src/components/common/SearchInput/SearchInput.tsx

```diff
@@ -12,7 +12,7 @@ import type {
   SearchInputRef,
 } from '@faststore/ui'
 import { MagnifyingGlass as MagnifyingGlassIcon } from 'phosphor-react'
-
+import useSearchHistory from 'src/hooks/useSeachHistory'
```

Comment:
don't add things to src/hooks. We already have the `sdk` folder for that. Either move everything from `sdk` to `hooks` or move this to `sdk`